src/main/java/org/elasticsearch/index/query/IdsQueryBuilder.java

```diff
@@ -88,6 +94,10 @@ public IdsQueryBuilder boost(float boost) {
         return this;
     }
 
+    public float getBoost() {
```

Comment:
one more thing, these are going to be user facing public methods, we should add javadocs for each of them